### PR TITLE
sync install steps with zh

### PR DESCRIPTION
**Pull request by windsonsea**
按中文版修改了 en 安装步骤

### CODE REVIEW

File: docs/en/install/install-dce.md [hunk 1]
```diff
@@ -6,18 +6,21 @@ This page describes the installation procedure of DCE 5.0.
...
 
     !!! note
 
-        - Available cluster resources: CPU > 10 cores, memory > 12 GB, remained storage space > 100 GB (currently, DCE 5.0 provides multiple replicas by default. If you choose a single replica, the resource consumption can be reduced to 2 cores and 4 GB)
-        - Kubernetes version: The latest stable version is recommended. It supports v1.21 to v1.24 currrently.
-        - Supported CRIs: Docker and containerd
-        - Storage: Prepare StorageClass and set it to the default. For details see [how to deploy a k8s cluster](install-k8s.md)
-        - Only support X86_64 currently
-        - Ensure your cluster has installed with CoreDNS
+        - Available cluster resources: CPU > 10 cores, memory > 12 GB, remained storage space > 100 GB (currently, DCE 5.0 provides multiple replicas by default. If you choose a single replica, the resource consumption can be reduced to 4 cores and 8 GB).
```
Comment (Michelle951) [inline docs/en/install/install-dce.md:17]:
```suggestion
        - Available cluster resources: CPU > 10 cores, memory > 12 GB, remained storage space > 100 GB (currently, DCE 5.0 provides multiple replicas by default. If you choose a single replica, the resource requirement can be reduced to 4 cores and 8 GB).
```

File: docs/en/install/install-dce.md [hunk 2]
```diff
@@ -6,18 +6,21 @@ This page describes the installation procedure of DCE 5.0.
...
+        - Kubernetes version: The latest stable version is recommended. It supports v1.20 to v1.24 currrently.
+        - Supported CRIs: Docker and containerd.
+        - Storage: Prepare StorageClass and set it to the default.
+        - Only support X86_64 currently.
+        - Ensure your cluster has installed with CoreDNS.
+        - If a cluster with a node, ensure you have removed the taints from the node.
```
Comment (Michelle951) [inline docs/en/install/install-dce.md:23]: "If a cluster with a node" 
A bit confusing, don't know what u mean if just read en version.

File: docs/en/install/offline-intall.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    ./dce5-installer install-app -c clusterConfig.yaml -p offline
+    ```
+    
+    !!! note
+
+        The flag `-p` specifies the air-gap folder to unzip the air-gap package.
+
+        For clusterConfig.yaml, see **step 2 of online installation** as above.
+
+6. After installation, you will get a successful tip on your screen. Congratulations! :smile: Now you can explore DCE 5.0 via the URL shown on your screen.
```
Comment (Michelle951) [inline docs/en/install/offline-intall.md:165]:
```suggestion
6. After installation, you will get a success tip on your screen. Congratulations! :smile: Now you can explore DCE 5.0 via the URL shown on your screen.
```

File: docs/en/install/install-dce.md (hunk 2 above)
Comment (windsonsea) [reply docs/en/install/install-dce.md]: Has changed to:

...removed the taint labels from ...

**Review by Michelle951 — APPROVED** [review]

**Pull request merged**